docs/source/usersguide/install.rst

```diff
@@ -113,34 +113,32 @@ Prerequisites
 
     * HDF5_ Library for portable binary output format
 
-      OpenMC uses HDF5 for binary output files. As such, you will need to have
-      HDF5 installed on your computer. The installed version will need to have
-      been compiled with the same compiler you intend to compile OpenMC with. If
-      you are using HDF5 in conjunction with MPI, we recommend that your HDF5
-      installation be built with parallel I/O features. An example of
-      configuring HDF5_ is listed below::
+      OpenMC uses HDF5 for many input/output files. As such, you will need to
+      have HDF5 installed on your computer. The installed version will need to
+      have been compiled with the same compiler you intend to compile OpenMC
+      with. On Debian derivatives, HDF5 and/or parallel HDF5 can be installed
+      through the APT package manager:
 
-           FC=/opt/mpich/3.1/bin/mpif90 CC=/opt/mpich/3.1/bin/mpicc \
-           ./configure --prefix=/opt/hdf5/1.8.12 --enable-fortran \
-                       --enable-fortran2003 --enable-parallel
+      .. code-block:: sh
 
-      You may omit ``--enable-parallel`` if you want to compile HDF5_ in serial.
+          sudo apt install libhdf5-dev
```

Comment:
Why cant the user install libhdf5-openmpi-dev (or mpich2) if they want hdf5 parallel support?  The docs seem to read like "if you want parallel support, you have to compile it yourself."   Also, do we not need hdf5-helpers?

Reply:
I realized that libhdf5-dev depends on hdf5-helpers, so it's not necessary to list that one too. Good point regarding the parallel versions of libhdf5-dev. I'll update this.